src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1514,14 +1516,27 @@ const deleteTwilioTransaction = async (
  * @returns ok(updated start page object) when update is successful
  * @returns err(FormNotFoundError) if form cannot be found
  * @returns err(PossibleDatabaseError) if start page update fails
+ * @returns err(InvalidPaymentAmountError) if payment amount exceeds MAX_PAYMENT_AMOUNT
  */
 export const updatePayments = (
   formId: string,
   newPayments: PaymentsUpdateDto,
 ): ResultAsync<
   IFormDocument['payments'],
-  PossibleDatabaseError | FormNotFoundError
+  PossibleDatabaseError | FormNotFoundError | InvalidPaymentAmountError
 > => {
+  const { amount_cents } = newPayments
+
+  // Check if payment amount exceeds maxPaymentAmountCents
+  if (amount_cents && amount_cents > paymentConfig.maxPaymentAmountCents) {
+    return errAsync(new InvalidPaymentAmountError())
+  }
+
+  // Check if payment amount is below minPaymentAmountCents
+  if (amount_cents && amount_cents < paymentConfig.minPaymentAmountCents) {
+    return errAsync(new InvalidPaymentAmountError())
+  }
```

Comment:
Should this be combined instead since the error thrown is the same? So like if `< min` or `> max`, then `return errAsync`